controllers/component_controller.go

```diff
@@ -475,8 +475,9 @@ func (r *ComponentReconciler) generateGitops(ctx context.Context, component *app
 
 	err = gitops.GenerateAndPush(tempDir, remoteURL, *component, r.Executor, r.AppFS, gitOpsBranch, gitOpsContext, gitopsConfig)
 	if err != nil {
-		log.Error(err, "unable to generate gitops resources due to error")
-		return err
+		gitOpsErr := util.SanitizeErrorMessage(err)
```

Comment:
needs to be also called on `RemoveAndPush()`

Reply:
Good catch

Reply:
Fixed